sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Transform.java

```diff
@@ -24,6 +24,8 @@
  * <p>
  * For example, the transform date(ts) is used to derive a date value from a timestamp column. The
  * transform name is "date" and its argument is a reference to the "ts" column.
+ *
+ * @since 3.0.0
  */
 @Experimental
```

Comment:
I think all DS v2 classes are still evolving. Shall we use `@Evolving` consistently? e.g. https://github.com/apache/spark/pull/27560/files#diff-ab2a72871faeb3ad8bae1d7f951899deR29

cc @brkyvz @rdblue